framework/include/interfaces/BlockRestrictable.h

```diff
@@ -224,6 +224,8 @@ class BlockRestrictable
    */
   Moose::CoordinateSystemType getBlockCoordSystem();
 
+  void isTheBlocksWithinDefaultBlocks() const;
```

Comment:
```suggestion
  void checkDefaultBlockConsistency() const;
```
or something like that